website/docs/client/java/java.md

```diff
@@ -10,7 +10,7 @@ The OpenLineage Java is a SDK for Java programming language that users can use t
 The core data structures currently offered by the client are the `RunEvent`, `RunState`, `Run`, `Job`, `Dataset`, 
 and `Transport` classes, along with various `Facets` that can come under run, job, and dataset.
 
-There are various [transport classes](#transports) that the library provides that carry the lineage events into various target endpoints (e.g. HTTP).
```

Comment:
instead of removing the anchor, can we consider linking out to https://openlineage.io/docs/client/java/configuration#transports?